src/keyring/ed25519/ledgertm.rs

```diff
@@ -0,0 +1,22 @@
+//! Ledger Tendermint signer
+
+use signatory::PublicKeyed;
+use signatory_ledger_tm::{self, Ed25519LedgerTmAppSigner};
+
+use crate::{
+    config::provider::ledgertm::LedgerTendermintConfig,
+    error::KmsError,
+    keyring::{ed25519::Signer, KeyRing},
+};
+
+pub const LEDGER_TM_PROVIDER_LABEL: &str = "ledgertm";
+pub const LEDGER_TM_ID: &str = "ledgertm";
```

Comment:
Shouldn't the ID come from the config? An ID that is always the same (and the same as the constant `LEDGER_TM_PROVIDER_LABEL`), feels wrong.